python/mlc_chat/compiler/convert_weight.py

```diff
@@ -62,11 +63,18 @@ def _calc_total_params(model: nn.Module) -> int:
 
 def _convert_args(args: ConversionArgs) -> None:  # pylint: disable=too-many-locals
     # model config & quantization config
+    with Target.from_device(args.device) as target:
+        extern.enable(
+            target=target,
+            flashinfer=False,
+            fastertransformer=args.quantization.kind == "ft-quant",
```

Comment:
`extern.enable` controls which external kernels to present specifically in compilation. And by design, quantization is assumed to compiler backend-independent, e.g. q4f16_1 works for all backends no matter it's cuda or not. In the case of `q4f16_ft`, even the compute kernel only exists on cuda, why does the quantization process depend on the existence of a compute kernel?